scss/screen.scss

```diff
@@ -1,372 +0,0 @@
-    display: inline-block;
-    background: lighten($dark-bg, 10%);
-    border-radius: 3px;
-    &:hover,
-    &:focus {
-      background: darken($dark-bg, 20%);
-      @include box-shadow(0 0 0 2px #FFF);
-    }
-  }
-}
-
-// nav in header
-#nav {
-  padding: 10px 0;
-  float: left;
-  a {
-    color: #FFF;
-    padding: 10px;
-    display: inline-block;
-    margin-right: -4px; // fix inline block spacing
-    &:link,
-    &:visited {
-      border: none;
-      border-right: 1px solid lighten($dark-bg, 10%);
-    }
-    &:hover,
-    &:active,
-    &:focus,
-    &.active {
-      background: lighten($dark-bg, 10%);
-    }
-    &:last-child {
-      border-right: none;
-    }
-  }
-}
-
-#banner {
-  // Cactus
-  background: #1E2A28 url(../images/open-refine-512px.png) center center;
-
-  @include background-size('cover');
-  margin: 0 0 20px;
-  padding: 20px 0;
-  position: relative;
-  &:after {
-    content: '';
-    width: 100%;
-    height: 100%;
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: #000;
-    background: rgba(0,0,0,0.5);
-  }
-  .inner-content {
-    @include container;
-  }
-  .copy {
-    position: relative;
-    z-index: 1;
-  }
-  h1 {
-    @include display-font;
-    font-size: 110px;
-    line-height: 0.9;
-    color: #FFF;
-  }
-  .intro {
-    font-size: 21px;
-    color: #FFF;
-    line-height: 1.33;
-    margin: 0;
-    @include opacity(0.75);
-  }
-
-  // Offset
-  .btn, .intro {
-    margin-left: 4px;
-  }
-  img {
-    margin-top: 20px;
-  }
-  .img-holder {
-    display: none;
-  }
-  @media all and (min-width: $break-large) {
-    &:after {
-      content: none;
-    }
-    height: 303px;
-    padding: 0;
-    .copy {
-      background: #000;
-      background: rgba(0,0,0,0.5);
-      padding: 20px;
-      margin-top: 40px;
-      @include span-columns(4,8);
-    }
-  }
-}
-
-#footer {
-  border-top: 1px solid $light-bg;
-  margin: 0 auto;
-  padding: 12px 0;
-  color: $muted-text;
-  overflow: hidden;
-  a {
-    @include muted-links;
-  }
-  @media all and (min-width: $break-large) {
-    p.first-child {
-      float: left;
-    }
-    p.last-child {
-      float: right;
-    }
-  }
-}
-
-// columns on home
-
-.cols {
-  margin: 0 0 20px;
-  overflow: hidden; // clearing
-  h3 {
-    margin: 12px 0 4px;
-  }
-  @media all and (min-width: 500px) {
-    .col {
-      @include span-columns(3);
-    }
-    .col:last-child {
-      @include omega;
-    }
-  }
-}
-
-// page with subnav
-
-.page-type-subnav {
-  #subnav {
-    h4 {
-      margin-top: 0;
-      margin-bottom: 4px;
-    }
-    ul {
-      padding: 0;
-      margin-top: 0;
-    }
-  }
-  #content {
-    article {
-      font-size: 14px;
-    }
-    h2 {
-      margin-top: 0;
-    }
-  }
-}
-
-@media all and (max-width: $break-medium - 1) {
-  #subnav {
-    padding: 12px 50px 0;
-  }
-  .page-type-subnav #content {
-    padding: 0 20px 12px 20px;
-  }
-}
-
-// Layout
-@media all and (min-width: $break-medium) {
```

Comment:
We should also setup a layout for larger screens (mine is 1920x1080) so that the .container is allowed to flex out wider or if I shrink by browser window then it goes back to minimum.  Look like the current width for the .container Box itself is too small by default on larger screens.  Look at some of the examples and everything on this page to use Flexbox more effectively https://css-tricks.com/snippets/css/a-guide-to-flexbox/#flexbox-examples

![capture](https://user-images.githubusercontent.com/986438/39079858-6ceb4a6e-44e8-11e8-8370-a8589c4aceaa.PNG)

Reply:
This theme doesn't have scss/screen.scss file.

Reply:
@thadguidry I suggest we merge the PR as is and open a new issue for the community to address this enhancement.

Reply:
@magdmartin Sure !

Reply:
#29 open